docs/user/dashboard/make-dashboards-interactive.asciidoc

```diff
@@ -177,7 +173,17 @@ Change the settings for the Options list and Range slider controls.
 
 . Hover over the control you want to edit, then click image:images/dashboard_controlsEditControl_8.3.0.png[The Edit control icon that opens the Edit control flyout].
 
-. On the *Edit control* flyout, change the options, then click *Save and close*.
+. On the *Edit control* flyout, change the options.
+
+. If you are editing an *Options list*, specify the additional settings:
+
+* To allow options to be inlcluded or excluded on the dashboard, select *Allow selections to be excluded*.
+
+* To allow an exists query to be created, select *Allow exists query*.
+
+* To allow the options to be sorted, select *Allow dynamic sorting of suggestions*, then select the default sort options. 
```

Comment:
These options [no longer exist](https://github.com/elastic/kibana/pull/147216) as of `8.7` - the only additional settings should be: 

![image](https://user-images.githubusercontent.com/8698078/220480022-a7573b69-9115-405e-aa69-fd67632937fb.png)